.azure-pipelines/vhdx.yml

```diff
@@ -0,0 +1,38 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger: none
+pr: none
+
+resources:
+  pipelines:
+    - pipeline: official
+      source: 'Azure.sonic-buildimage.official.vs'
+      trigger:
+        branches:
+        - master
+  containers:
+    - container: slave
+      image: sonicdev-microsoft.azurecr.io/sonic-slave-buster:latest
+
+jobs:
+- job:
+  displayName: 'Build VHDX image'
+  pool:
+    vmImage: 'ubuntu-20.04'
+  container: slave
+  steps:
+    - checkout: none
+    - download: official
+      artifact: sonic-buildimage.vs
+      patterns: 'target/sonic-vs.img.gz'
+    - script: |
+        set -ex
+        cd ../official/sonic-buildimage.vs/target/
+        gzip -d sonic-vs.img.gz
```

Comment:
Can we add the image in the vs official build?

Reply:
I am not sure if it is enough. The old job is more complex.
The concern is maybe we will add a long if codes in that build script.